### PR TITLE
Highest att slashing protection + far future protection

### PR DESCRIPTION
**Pull request by alonmuroch**

### CODE REVIEW

**Review by roman-blox — APPROVED** [review]

File: core/networks.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+func (n Network) FullPath(relativePath string) string {
+	return BaseEIP2334Path + relativePath
+}
+
+func (n Network) MinGenesisTime() uint64 {
+	switch n {
+	case PyrmontNetwork:
+		return 1605700800
+	case MainNetwork:
+		return 1606824000
```
Comment (olegshmuelov) [inline core/networks.go:60]: should the genesis time reflect the delay? 23 sec for mainnet and 7 for pyrmont

**Review by olegshmuelov — APPROVED** [review]

File: core/networks.go (hunk 1 above)
Comment (alonmuroch) [reply core/networks.go:60]: which delay you mean? I took those numbers from prysm

**Comment by roman-blox** [comment]
- Merge conflicts should be resolved
- Tests are failed

File: core/networks.go (hunk 1 above)
Comment (olegshmuelov) [reply core/networks.go:60]: I mean that MainNet Genesis is at 2020-12-01 12:00:23 UTC and not at 2020-12-01 12:00:00 UTC as calculated from 1606824000

Comment (alonmuroch) [reply core/networks.go]: fixed for both

**Pull request merged**